packages/apps-v5/src/commands/ps/index.js

```diff
@@ -30,8 +30,8 @@ function printExtended (dynos) {
   })
 }
 
-async function printAccountQuota(context, heroku, app, account) {
-  if (app.process_tier !== 'free' && app.process_tier !== 'eco') {
+async function printAccountQuota (context, heroku, app, account) {
+  if (app.process_tier !== 'eco') {
```

Comment:
Since we will still have free dynos in the system for some time, we need to continue to handle them, so this should keep the free check.